data/widgets/banner/dynamic.ui

```diff
@@ -6,26 +6,33 @@
     <property name="visible">True</property>
     <property name="can_focus">False</property>
     <child>
-      <object class="EknDynamicLogo" id="logo">
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
-        <property name="valign">end</property>
-        <property name="expand">True</property>
-        <property name="image-uri">resource:///app/assets/logo</property>
+      <object class="GtkGrid" id="grid">
```

Comment:
Since we have a custom `vfunc_draw` in EknDynamicLogo already, I'd suggest adding a `Gtk.render_background()` there instead.